src/main/java/bio/terra/cda/app/generators/SqlGenerator.java

```diff
@@ -61,29 +60,29 @@ public SqlGenerator(String qualifiedTable, Query rootQuery, String version, bool
     this.project = dotPos == -1 ? "" : qualifiedTable.substring(0, dotPos);
     this.table = dotPos == -1 ? qualifiedTable : qualifiedTable.substring(dotPos + 1);
     this.fileTable =
-        dotPos == -1
-            ? qualifiedTable.replace("Subjects", TableSchema.FILES_COLUMN)
-            : qualifiedTable.substring(dotPos + 1).replace("Subjects", TableSchema.FILES_COLUMN);
+            dotPos == -1
+                    ? qualifiedTable.replace("Subjects", TableSchema.FILES_COLUMN)
+                    : qualifiedTable.substring(dotPos + 1).replace("Subjects", TableSchema.FILES_COLUMN);
     this.tableDefinition = TableSchema.getSchema(version);
     this.dataSetInfo =
-        new DataSetInfo.DataSetInfoBuilder().addTableSchema(version, this.tableDefinition).build();
-
+            new DataSetInfo.DataSetInfoBuilder().addTableSchema(version, this.tableDefinition).build();
     preInit();
     initializeEntityFields();
     initializeBuilders();
   }
 
   public SqlGenerator(
-      String qualifiedTable,
-      Query rootQuery,
-      String version,
-      boolean filesQuery,
-      ParameterBuilder parameterBuilder,
-      ViewListBuilder<View, ViewBuilder> viewListBuilder)
-      throws IOException {
+          String qualifiedTable,
+          Query rootQuery,
+          String version,
+          boolean filesQuery,
+          ParameterBuilder parameterBuilder,
+          ViewListBuilder<View, ViewBuilder> viewListBuilder)
+          throws IOException {
```

Comment:
can you fix the formatting of this file so we don't have whitespace differences? it's hard to see what has actually changed.